src/pseudopeople/utilities.py

```diff
@@ -81,22 +76,20 @@ def get_index_to_noise(
 
     index_eligible_for_noise = dataset.get_non_empty_index(required_columns)
 
-    # As long as noise is relatively rare, it will be faster to randomly select cells to
-    # noise rather than generating a random draw for every item eligible
-    if isinstance(noise_level, float) and noise_level < 0.2:
-        rng = np.random.default_rng(
-            seed=get_hash(f"{dataset.randomness.seed}_get_index_to_noise_{additional_key}")
+    if isinstance(noise_level, float):
+        number_to_noise = dataset.randomness.binomial(
```

Comment:
This confused me. You set up a np.random.default_rng but then changed this approach from using it to instead using dataset.randomness. Was that just functionaltiy that was added to `dataset` later and this never got updated?

Reply:
Yes. This function was creating a default_rng instance but we already create that with our Dataset class now so it is unnecessary to make a new instance in any of the utility functions.